scripts/multiply_by_x.py

```diff
@@ -7,10 +10,14 @@ def main(params: Parameters):
     input_file_path = params.existing_file("input_file")
     output_file_path = params.creatable_file("output_file")
     x = params.integer("x")
+    logging.info("Reading from input file: %s", str(input_file_path.absolute()))
     with input_file_path.open() as input_file:
-        with output_file_path.open() as output_file:
+        with open(str(output_file_path.absolute()), "w") as output_file:
```

Comment:
You just want to pass "w" to `.open()` as it was previously written (it is the more idiomatic way of opening `pathlib` `Path`s)